torchx/schedulers/ray_scheduler.py

```diff
@@ -397,7 +397,17 @@ def log_iter(
             return iterator
 
         def list(self) -> List[str]:
-            raise NotImplementedError()
+            address = os.getenv("RAY_ADDRESS")
+            if not address:
+                raise Exception(
+                    "RAY_ADDRESS env variable is expected to be set validly to list jobs on ray scheduler."
```

Comment:
"validly" reads weird, what about:
expecting valid RAY_ADDRESS env variable to be set to list jobs on ray scheduler.